### PR TITLE
demo: show how to avoid <cleartext HTTP traffic not permitted> an…

### LINKED ISSUE #1171: 缺少android.permission.FOREGROUND_SERVICE权限
如题。
FiledownloaderService.makeServiceForeground方法需要用到FOREGROUND_SERVICE权限，AndroidManifest.xml中没有声明

### PR DESCRIPTION
**Pull request by rantianhua**
…d lack of FOREGROUND_SERVICE permission in demo

more detail please refer to https://stackoverflow.com/questions/45940861/android-8-cleartext-http-traffic-not-permitted

Closes #1171

### CODE REVIEW

**Comment by Jacksgong** [comment]
I can't get point about cleartext HTTP traffic not permitted. could you please more detail about that refer to the commit you provide, and if it will affect the real case if people use FileDownloader, please add it to the readme doc.

p.s. change the MR title from `refactor` to `demo`.

**Comment by Jacksgong** [comment]
Okay, what about other URLs declared on the [Contacts](https://github.com/lingochamp/FileDownloader/blob/master/demo/src/main/java/com/liulishuo/filedownloader/demo/Constant.java)?

**Comment by rantianhua** [comment]
This is a sample for solving this issue in demo, so there is no need to cover all URLs in [Constant](https://github.com/lingochamp/FileDownloader/blob/master/demo/src/main/java/com/liulishuo/filedownloader/demo/Constant.java).

**Comment by Jacksgong** [comment]
赞。

File: README.md [hunk 1]
```diff
@@ -63,6 +64,21 @@ From now on, FileDownloader support following components to be customized by you
 
 Just create your own `DownloadMgrInitialParams.InitCustomMaker` and put those customized component to it, finally init the FileDownloader with it: [FileDownloader#init](https://github.com/lingochamp/FileDownloader/blob/master/library/src/main/java/com/liulishuo/filedownloader/FileDownloader.java#L62)
 
+## Adaptation
```
Comment (Jacksgong) [inline README.md]: @rantianhua 中文的readme中要不也加下?

Comment (rantianhua) [reply README.md]: 有的，你应该看到了吧？

**Pull request merged**